README.md

```diff
@@ -17,8 +17,8 @@ process that you can fine-tune to your needs.
 - Feast your eyes on [lots of working examples][examples].
 - Our CI checks every contribution for regressions: ![Build Status](https://circleci.com/gh/modm-io/modm/tree/develop.svg?style=shield)
```

Comment:
Let's also remove this badge, since develop will have a different CI setup making this useless.

Reply:
Ok